### ISSUE TITLE
Errors when installing in a folder with "."

### ISSUE THREAD

**Issue by DylanGuedes** [kerl/kerl#108]
Hello everyone,

After installing erlang with kerl using "kerl install 18.2 ~/.erlang", the installation worked fine but everytime that I interact with erlang I get an error, because of the folder name. The error is:
```
file:path_eval([".","/home/dguedes"],".erlang"): illegal operation on a directory
```
After installing in a folder without dot (".") in its name, everything worked fine. 

cheers.

**Comment by mrallen1** [comment]
This ought to work. We need to figure out what's going wrong and why.

**Comment by mrallen1** [comment]
OK, so after some digging around with @sanmiguel it turns out that `~/.erlang` is a special file that the `erl` command consults on start up. So basically the answer here is unfortunately, *don't do this* - choose a different dot-directory name like `~/.erlangbuilds` or something.  

We are going to add a check into kerl that warns people about this issue because it's somewhat poorly documented or at least not well known. 

For details about the `erl` startup file options, you can read that [here](http://erlang.org/doc/man/erl.html) (See the section about `Configuration`)

**Comment by DylanGuedes** [comment]
Thank you, that's my bad so.

**Comment by mrallen1** [comment]
We just merged #127 which should prevent this accidental mistake in the future. Thanks!
